### PR TITLE
Update chinese simplified and chinese traditional files for 2.22.1

### PR DESCRIPTION
**Pull request by TomStilson**

### CODE REVIEW

File: locales/zh-tw.json [hunk 1]
```diff
@@ -83,6 +83,7 @@
   "evidence_cannot_take": "階級不夠，無法拿取證物",
   "dumpster": "垃圾桶",
   "crafting_item": "製作 %s",
-  "crafting_bench": "工作台",
+  "crafting_bench": "工作臺",
```
Comment (FAr18) [inline locales/zh-tw.json:86]:
```suggestion
  "crafting_bench": "工作台",
```

File: locales/zh-tw.json [hunk 2]
```diff
@@ -83,6 +83,7 @@
...
+  "open_crafting_bench": "打開工作臺",
```
Comment (FAr18) [inline locales/zh-tw.json:87]:
```suggestion
  "open_crafting_bench": "打開工作台",
```

**Review by FAr18 — CHANGES_REQUESTED** [review]
"台" or "檯" should be used. And I think it's better to use fewer strokes between synonyms.

File: locales/zh-tw.json (hunk 1 above)
Comment (TomStilson) [reply locales/zh-tw.json]: Thank you for your advice, will update now.

**Pull request merged**